ngraph/src/ngraph/node.cpp

```diff
@@ -124,29 +124,14 @@ std::shared_ptr<Node>
     return clone;
 }
 
-std::shared_ptr<Node> Node::copy_with_new_args(const NodeVector& args) const
-{
-    NODE_VALIDATION_CHECK(
-        this, false, "Internal error: copy_with_new_args not replaced by clone_with_new_inputs");
-}
-
 std::shared_ptr<Node> Node::clone_with_new_inputs(const OutputVector& inputs) const
 {
-    NodeVector args;
-    for (const Output<Node>& input : inputs)
-    {
-        args.push_back(get_output_element(input));
-    }
-    std::shared_ptr<Node> clone = copy_with_new_args(args);
-    // Remove the inserted GOEs
-    for (size_t i = 0; i < inputs.size(); ++i)
-    {
-        if (clone->input_value(i) != inputs.at(i))
-        {
-            clone->set_argument(i, inputs.at(i));
-        }
-    }
-    return clone;
+    NODE_VALIDATION_CHECK(this,
+                          false,
+                          "Internal error: clone_with_new_inputs not implemented for node ",
+                          get_friendly_name(),
+                          " with type ",
+                          description());
```

Comment:
Why you removed `clone_with_new_inputs` implementation for `ngraph::Node`?

Reply:
`clone_with_new_inputs` is deprecated method which cannot work with operations which have several outputs.

Reply:
If `clone_with_new_inputs` is deprecated why you replace `copy_with_new_args` with it?

Reply:
Sorry my mistake. `copy_with_new_args` is deprecated, `clone_with_new_inputs` is a right method.